openassessment/xblock/submission_mixin.py

```diff
@@ -738,9 +740,21 @@ def submission_path_and_context(self):
 
         file_urls = None
 
+        team_id_for_shared_files = None
```

Comment:
Maybe it has to wait until we consolidate the stories in flight but I'd argue for a different naming. This suggests to me that we have different team IDs for files and other parts of the page when I thought we wanted to source most of the page context differently based on whether the team has submitted or not. 
```suggestion
        submission_team_id = None
```

Reply:
+1